### ISSUE TITLE
[FEATURE] Provide an option to load private prompts from Langchain Hub

### ISSUE THREAD

**Issue by ghane100** [FlowiseAI/Flowise#1966]
Currently the Langchain Hub option in Prompt Template or Chat Prompt Template are displaying only prompts that are publicly available. Is it possible to provide an option with some authentication or by using some kind of keys to access prompts which are created by Langsmith users and which are private.

**Comment by dentro-innovation** [comment]
yes allowing users to use their private prompts would be very much appreciated!

I'd also add the request for a better search in the langchain hub, i.e. allow me to search for keywords instead of only the dropdown boxes
